docs/index.md

```diff
@@ -97,11 +97,7 @@ See our [documentation](https://dockstarter.com/introduction/) for more detailed
 
 ## Support
 
-[![Discord chat](https://img.shields.io/discord/477959324183035936.svg?style=flat-square&color=607D8B&logo=discord)](https://discord.gg/YFyJpmH)
-
-Click the chat badge to join us on Discord for support!
-
-[[Feature Request](https://github.com/GhostWriters/DockSTARTer/issues/new?template=feature_request.md)] [[Bug Report](https://github.com/GhostWriters/DockSTARTer/issues/new?template=bug_report.md)]
+See our [Support Page](basics/support.md)
```

Comment:
This page is also the readme in the main repo. I'd like to keep this info visible. We can still add the link though.